packages/compiler-core/src/parser.ts

```diff
@@ -634,7 +634,7 @@ function onCloseTag(el: ElementNode, end: number, isImplied = false) {
   if (!inVPre) {
     if (tag === 'slot') {
       el.tagType = ElementTypes.SLOT
-    } else if (isFragmentTemplate(el)) {
+    } else if (isFragmentTemplate(el, stack[0])) {
```

Comment:
`stack` is in global scope and no need to be passed in.

Reply:
thanks fixed.